models/mapper_test.go

```diff
@@ -63,8 +90,44 @@ func TestMapZebedeeDataToSearchDataImport(t *testing.T) {
 				CanonicalTopic:  canonicalTopic,
 			},
 		}
+		Convey("When mapped with a default keywords limit when edition is present", func() {
+			result := models.MapZebedeeDataToSearchDataImport(zebedeeDataWithEdition, -1)
+			Convey("Then the result should be validly mapped with 4 keywords when edition is present", func() {
+				So(result.UID, ShouldResemble, someTitle+someEdition)
+				So(result.DataType, ShouldResemble, someRelease)
+				So(result.CDID, ShouldResemble, someCDID)
+				So(result.DatasetID, ShouldResemble, someDatasetID)
+				So(result.MetaDescription, ShouldResemble, someMetaDescription)
```

Comment:
It is probably worth adding assertion for `result.Edition`

Reply:
done in update 1.